### DISCUSSION TITLE
Language selection German, some fields/responses are not translated

### DISCUSSION THREAD

**Discussion by nomemolesto** [microsoft-search/pnp-modern-search#2099] [Ideas]
**Is your feature request related to a problem? Please describe.**
We use PNPMS with ch-de and de-de browers. Some fields are not translated.

**Describe the solution you'd like**
We would love all fields are translated when language is selected.

**Describe alternatives you've considered**
If you like we could help you in translation to GE.

**Additional context**
Screenshot of current GUI, missing translated fields are highlighted.
![GUI PNPMS German translation missing](https://user-images.githubusercontent.com/57444142/169284769-a6a2cbdf-4b1c-4841-80d3-9f204c51e62a.png)

**Comment by FranckyC** [comment]
Hi, what version are you using? Also, is it the same with ch-de and de-de languages?

**Reply by nomemolesto** [reply]
We are using version 4.6.1.
ch-de and de-de is the same, yes. In Switzerland only few words are different to Germany-German.

**Reply by FranckyC** [reply]
Ok that is weird because the labels seems to be there. Did you spot any other missing translations by any chance?

**Reply by nomemolesto** [reply]
We use it on two independent tenants, both ch-de. Only missing is the "results" instead of "Resultat" (singular) of "Resultate" (plural) and the "Select value" in all filters (Wert auswählen).

**Reply by FranckyC** [reply]
When you ch-de, where does come this information? User preferred language, browser language, site language?

**Reply by nomemolesto** [reply]
Probably not from the browser (mine is set to German, German Switzerland, German Germany) but from the MyAccount settings of the M365 user account. Mine is set to preferred Swiss German (ch-de).

**Reply by wobba** [reply]
If you remove language preference from the profile settings in UPA and M365 profile, then you can easily test by just changing the browser language. https://www.techmikael.com/2018/02/testing-multiple-languages-in.html

**Reply by GerritGoudsmit** [reply]
Thanx for your help, after setting the language preferences and updating the modern search, everything was displayed in the right language.
